README.md

```diff
@@ -40,34 +40,61 @@ Install the migrations:
 $ rake alchemy_spree:install:migrations
 ```
 
-Migrate the database:
+Do the next step before running the migrations.
 
-```sh
-$ rake db:migrate
-```
 
 ### Authentication system installation
 
-Since Alchemy 3.0 has dropped the authentication from its core, you will need to choose one authentication system. The easiest choice is to use the [alchemy-devise gem](https://github.com/magiclabs/alchemy-devise).
+Since Alchemy 3.0 has dropped the authentication from its core, you will need to choose an  authentication system yourself. There are 2 available options:
```

Comment:
> Since Alchemy 3.0 has dropped the authentication from its core, you will need to choose an  authentication system yourself. There are 2 available options:

Not only Alchemy has dropped the authentication from it's core, Spree has also dropped it. And they did to support bringing your own authentication and that's the third option the developer has: 

1. Using `Spree::Auth::Devise`
2. Using `Alchemy::Devise`
3. Build their own authentication

If they choose to use `Spree::Auth::User` they have to tell Alchemy the user class.
If they choose `Alchemy::User` they have to tell Spree the user class. 
And if they choose to build their own, they have to tell Alchemy and Spree the user class.